odoo/tools/misc.py

```diff
@@ -1814,7 +1814,7 @@ def verify_hash_signed(env, scope, payload):
     return None
 
 
-def limited_field_access_token(record, field_name, timestamp=None):
+def limited_field_access_token(record, field_name, *, scope, timestamp=None):
```

Comment:
Should add a case in the tests to ensure different scope = different token + validation doesn't work if the token is not given for the right scope.

Reply:
Scope is directly passed to the hmac function, I don't think this deserves a test.

Reply:
It deserves a test to make sure it is correctly passed. This is low level security stuff, we don't want to leave a chance.